src/main/resources/db_scripts/scheduled/expired-reservations-clean-up.sql

```diff
@@ -0,0 +1,5 @@
+/* Clean up expired reservations */
+
+DELETE FROM event_bookings
```

Comment:
I suggest we change this to updating the record to CANCELLED but we first need to change the API so that cancelled bookings don't count towards the teacher's reservation quota.